openfl-tutorials/interactive_api_tutorials_(experimental)/Pytorch_TinyImageNet_workspace/new_python_api_tinyimagenet.ipynb

```diff
@@ -82,7 +82,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "optimizer_adam = optim.Adam(model_net.parameters(), lr=1e-4)\n",
+    "params_to_update = []\n",
+    "for name,param in model_net.named_parameters():\n",
```

Comment:
`.parameters()` iterator would be enough here since parameters names are not used anywhere.